src/test/ui/issue-45157.stderr

```diff
@@ -7,15 +7,6 @@ LL |         let mref = &mut u.s.a;
 LL |         let nref = &u.z.c;
    |                    ^^^^^^ immutable borrow occurs here
 
-error[E0502]: cannot borrow `u.s.a` as mutable because it is also borrowed as immutable
```

Comment:
Hmm I *think* this is okay that this error went away, but I'll want to double check